src/eddymotion/dmri.py

```diff
@@ -211,35 +199,51 @@ def from_filename(cls, filename):
 
 
 def load(
-    filename, gradients_file=None, b0_file=None, brainmask_file=None, fmap_file=None
+    filename,
+    gradients_file=None,
+    b0_file=None,
+    brainmask_file=None,
+    fmap_file=None,
+    bvec_file=None,
+    bval_file=None,
+    b0_thres=50,
 ):
     """Load DWI data."""
     filename = Path(filename)
     if filename.name.endswith(".h5"):
         return DWI.from_filename(filename)
 
-    if not gradients_file:
+    if gradients_file:
```

Comment:
Want to add some error handling here in case someone foolishly provides both kinds of input?

Reply:
Added a warning